server/server_test.go

```diff
@@ -1322,12 +1324,28 @@ func startTestServer(config string) error {
 	if err != nil {
 		return err
 	}
+
 	go func() {
 		err := server.Start()
 		if err != nil {
 			return
 		}
 	}()
+
+	retry := 3
```

Comment:
Moving a well named constant used once over 1300 LOC up seems like micro overkill to me.